lib/input/azure_queue_storage.go

```diff
@@ -79,6 +79,9 @@ func (a *azureQueueStorage) ReadWithContext(ctx context.Context) (msg types.Mess
 			}
 			msgIDURL := messageURL.NewMessageIDURL(queueMsg.ID)
 			_, err = msgIDURL.Delete(ctx, queueMsg.PopReceipt)
```

Comment:
Would this `Delete` call effectively be an acknowledgement? If so this ought to be placed within the ack func that gets returned below.

Reply:
Hi! This one's on me, my bad.
Hopefully, #689 will fix it.

Reply:
Awesome, thanks for the quick investigation! Happy to drop this change and rebase once that's merged.

Reply:
Cool, thank you @mihaitodor

Reply:
Rebase done (cc @Jeffail)